src/fn/fn-link.scss

```diff
@@ -0,0 +1,11 @@
+@import "./fn-settings";
+
+$block: #{$fn-namespace}-link;
```

Comment:
if this is a separate component we should have a page for it, with all states that can be applied: hover, active, disabled... so we can detect future changes. The cursor is not set and the underline is missing.

Reply:
i will skip the doc page for now. I will add the rest

Reply:
done